src/es/leercapitulo/src/eu/kanade/tachiyomi/extension/es/leercapitulo/LeerCapitulo.kt

```diff
@@ -33,10 +33,12 @@ class LeerCapitulo : ParsedHttpSource() {
 
     override val baseUrl = "https://www.leercapitulo.co"
 
-    override val client = super.client.newBuilder()
+    override val client = network.cloudflareClient.newBuilder()
         .rateLimitHost(baseUrl.toHttpUrl(), 1, 3)
         .build()
 
+    private val notRateLimitClient = network.cloudflareClient.newBuilder().build()
```

Comment:
```suggestion
    private val notRateLimitClient = network.cloudflareClient
```

Should be the same unless I'm missing some subtle difference.